apps/engineeringblog/app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import ThemeRegistry from "@/theme/ThemeRegistry";
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
```

Comment:
If we're using Material UI, why do we need `global.css`?

Reply:
I was thinking it would be used to import global css styles like `animate.css` should we decide to use something like that.